static/css/components/modal.css

```
@@ -86,12 +86,13 @@
    meant for regular low-risk actions
    such as exiting */
 .modal-btn-secondary {
-  background-color: var(--secondary-bg);
-  color: var(--ink);
+  background-color: var(--secondary-btn-bg);
+  
+  color: #FFFFFF;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)